### PR TITLE
Update deathplane reset, add Nasa at end of big city

### PR DESCRIPTION
**Pull request by fheysel**

### CODE REVIEW

File: godotproject/DeathPlane.gd [hunk 1]
```diff
@@ -1,6 +1,17 @@
 extends Area2D
 
+const level0_start_postion :=  Vector2(121, 896)
+const level1_start_postion := Vector2(121, 256)
```
Comment (charasyn) [inline godotproject/DeathPlane.gd]: Can we change this to be:
```suggestion
export (Vector2) var start_position
```
and then set it inside each Level scene?

File: godotproject/DeathPlane.gd [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
-		body.takeDamage(1);
-		Global.fade_to_scene("res://Level" + str(int(get_tree().current_scene.name)) + ".tscn")
+		level_name = Global.get_current_scene_property('level_name')
+		if level_name == "Morning Meadow":
+			Global.fade_set_body_to_position(body, level0_start_postion)
+		elif level_name == "The Big City":
+			Global.fade_set_body_to_position(body, level1_start_postion)
+		
+		body.takeDamage(1)
+		# Global.fade_to_scene("res://Level" + str(int(get_tree().current_scene.name)) + ".tscn")
```
Comment (charasyn) [inline godotproject/DeathPlane.gd]: Remove commented code
```suggestion
```

**Review by charasyn — CHANGES_REQUESTED** [review]
Hey just one major thing about the start position. I think it should be put as an `export var` so that we can set it from the GUI differently for each level.
After making it an export var, you can go to the editor, select the death plane in each level, and adjust the start position on the right hand side in the property editor

File: godotproject/DeathPlane.gd (hunk 1 above)
Comment (fheysel) [reply godotproject/DeathPlane.gd]: sounds good

File: godotproject/Level0.tscn [hunk 3]
```diff
@@ -33,6 +33,8 @@ margin = 10.0
 player_nodepath = NodePath("../Player")
 
 [node name="DeathPlane" parent="." instance=ExtResource( 4 )]
+level0_start_postion = Vector2( 121, 896 )
+level1_start_postion = Vector2( 121, 256 )
```
Comment (charasyn) [inline godotproject/Level0.tscn]: In Level0.tscn, set start_position to the correct start position for level 0 (121, 896)
```suggestion
[node name="DeathPlane" parent="." instance=ExtResource( 4 )]
start_position = Vector2( 121, 896 )
```

File: godotproject/Level1.tscn [hunk 4]
```diff
@@ -352,6 +352,8 @@ player_nodepath = NodePath("../Player")
 
 [node name="DeathPlane" parent="." instance=ExtResource( 9 )]
 position = Vector2( -56, -680 )
+level0_start_postion = Vector2( 121, 896 )
+level1_start_postion = Vector2( 121, 256 )
```
Comment (charasyn) [inline godotproject/Level1.tscn]: In Level1.tscn, set the start_position to the correct value for Level 1 (121, 256)
```suggestion
[node name="DeathPlane" parent="." instance=ExtResource( 9 )]
position = Vector2( -56, -680 )
start_position = Vector2( 121, 256 )
```

File: godotproject/DeathPlane.gd [hunk 5]
```diff
@@ -1,7 +1,9 @@
 extends Area2D
 
-const level0_start_postion :=  Vector2(121, 896)
-const level1_start_postion := Vector2(121, 256)
+export (Vector2) var level0_start_postion
+export (Vector2) var level1_start_postion
+#const level0_start_postion :=  Vector2(121, 896)
+#const level1_start_postion := Vector2(121, 256)
```
Comment (charasyn) [inline godotproject/DeathPlane.gd]:
```suggestion
export (Vector2) var start_position
```

File: godotproject/DeathPlane.gd [hunk 6]
```diff
@@ -14,4 +16,4 @@ func _on_DeathPlane_body_entered(body):
 			Global.fade_set_body_to_position(body, level1_start_postion)
```
Comment (charasyn) [inline godotproject/DeathPlane.gd]:
```suggestion
			Global.fade_set_body_to_position(body, start_position)
```
We should only need one `start_position` variable and remove the `if` statement here

**Review by charasyn — CHANGES_REQUESTED** [review]
That's not quite what I meant. I think we should have one variable `start_position` and set it differently in the scenes, and then we don't need to check which level we're in because we can just go to the `start_position` that we're told

**Comment by fheysel** [comment]
ohhh i see, I didnt realize we could do that

**Review by charasyn — APPROVED** [review]
Awesome! 👍

**Pull request merged**